.travis.yml

```diff
@@ -1,5 +1,15 @@
+sudo: false
+
 language: node_js
+
 node_js:
-  - "stable"
-  - "5"
-  - "4"
+  - 6
+  - 4
+
+cache:
+  directories:
+    - node_modules
+
+install:
+  - npm i -g npm@latest
```

Comment:
This seems a bit unsafe to me. Each node comes with its own npm, why should we switch to latest node?